cmd/commands/utils.go

```diff
@@ -25,14 +26,14 @@ import (
 
 //This variable can be overwritten by plugin applications
 // if they require a different working directory
-var DefaultHome = "basecoin"
+var DefaultHome = ".basecoin"
```

Comment:
This variable exists because it is changed by each basecoin plugin which may require a modified genesis (and the original space should not be overwritten). Only if we choose not to enforce use of the home directory for default basecoin-plugin working spaces should be we change this. If we choose to enforce using the home directory (which I had assumed we would) then it does not make sense to modify this to include the full path as it would mean that each plugin would be duplicating the code `os.Getenv("Home")`.. Just considering